static/setup_secure_permissions_nextcloud.sh

```diff
@@ -45,14 +45,16 @@ fi
 
 print_text_in_color "$ICyan" "chmod Files and Directories"
 find "${NCPATH}"/ -type f -print0 | xargs -0 chmod 0640
-find "${VMLOGS}"/nextcloud.log -type f -print0 | xargs -0 chmod 0640
 find "${VMLOGS}"/audit.log -type f -print0 | xargs -0 chmod 0640
 find "${NCPATH}"/ -type d -print0 | xargs -0 chmod 0750
+find "${VMLOGS}"/ -type d -print0 | xargs -0 chmod 0750
+find "${VMLOGS}"/nextcloud.log -type f -print0 | xargs -0 chmod 0640
 
 print_text_in_color "$ICyan" "chown Directories"
+chown "${htuser}":"${htgroup}" "${VMLOGS}"/
+chown "${htuser}":"${htgroup}" "${VMLOGS}"/nextcloud.log
+chown "${htuser}":"${htgroup}" "${VMLOGS}"/audit.log
```

Comment:
But I am not sure if you need to chown these files explicitly, that are contained in the vmlogs-folder, if you chown the whole folder...

Reply:
There are files in there which are root:root which I want to stay as root:root. Hence, I chown the dir itself, and the files needed for it to work.